### ISSUE TITLE
Show time zone for scheduled push notifications

### ISSUE THREAD

**Issue by david-venhoff** [digitalfabrik/integreat-cms#2371]
### Motivation

When working across different timezones, it could be confusing what the timezone of the scheduled send date for push notifications is (The timezone of the user or the timezone of the region of the push notification?).

### Proposed Solution

Show the used time zone in the push notification form and the push notification list view

### Alternatives

Either use the user time zone or the push notification region time zone and make that clear to the user

### User Story

As a user of the cms I would like to know when I have scheduled a push notification for a region that is in a different time zone.

**Comment by osmers** [comment]
I don't think this happens very often, so I will close it for now, as we have also never received requests about it afaict. We can reopen if the need becomes apparent.
